### ISSUE TITLE
center_of_mass() fails for one of its examples

### ISSUE THREAD

**Issue by prjemian** [bluesky/bluesky#1794]
In `bluesky.callbacks.fitting.center_of_mass()`, there is an Example in the comments: https://github.com/bluesky/bluesky/blob/997eb2289420da244ba243b65816611403931273/src/bluesky/callbacks/fitting.py#L158-L166

The example shows a call to `scipy.ndimage.measurements.center_of_mass(b, lbl, [1,2])` and this runs successfully on the example array.  When the same data submitted to `bluesky.callbacks.fitting.center_of_mass(b, lbl, [1,2])`, this exception is raised: `TypeError: Field elements must be 2- or 3-tuples, got '1'`

Note this comment in preceding the `center_of_mass()` function in the source code: https://github.com/bluesky/bluesky/blob/997eb2289420da244ba243b65816611403931273/src/bluesky/callbacks/fitting.py#L128-L129

## Expected Behavior
The example code should be correct.  The example code should be an example of the  function in this module, not from `scipy`.

**Comment by tacaswell** [comment]
I think we should make this private, strip the extra (non-functional) args, and delete the docstring.

**Comment by prjemian** [comment]
I agree with this.  It's use is limited and should not be part of the public API.
